src/features/settings/pages/SecurityPage.tsx

```
@@ -97,490 +91,320 @@ export interface SecurityPageProps {
-        description={t('settings:security.logout.all.description')}
-      >
-        <SettingsDangerButton
-          size="sm"
-          onClick={() => handleLogout('all')}
-        >
-          {t('settings:security.logout.all.button')}
-        </SettingsDangerButton>
-      </SettingRow>
+      <section className="mt-8 rounded-[20px] border border-[color:color-mix(in_oklab,var(--neg)_30%,var(--rule))] bg-[color:color-mix(in_oklab,var(--neg)_6%,var(--card))] px-5 py-5 sm:px-6">
+        <h3 className="font-serif text-2xl font-normal tracking-tight text-[var(--neg)]">Delete account</h3>
+        <p className="mt-1 max-w-[60ch] text-[13.5px] leading-relaxed text-ink-2">Permanently delete your Blawby account and all associated data. This removes you from the organization but does not delete the practice. Transfer ownership first if you're the sole owner.</p>
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&`#39`;`, `&rsquo;`.

(react/no-unescaped-entities)